astyanax-thrift/src/main/java/com/netflix/astyanax/thrift/ThriftCqlFactoryResolver.java

```diff
@@ -6,7 +6,7 @@
 import com.netflix.astyanax.AstyanaxConfiguration;
 
 public class ThriftCqlFactoryResolver {
-    private static final Pattern VERSION_REGEX = Pattern.compile("^([0-9])+\\.([0-9])+(.*)");
+    private static final Pattern VERSION_REGEX = Pattern.compile("^([0-9])++\\.([0-9])++(.*)+");
```

Comment:
are there any tests that verify that no regressions are introduced?

Reply:
I assume that the [ThriftKeyspaceAllRowsTest](https://github.com/bazaarvoice/astyanax/blob/master/astyanax-test/src/test/java/com/netflix/astyanax/thrift/ThriftKeyspaceAllRowsTest.java) and [ThriftKeyspaceImplTest](https://github.com/bazaarvoice/astyanax/blob/master/astyanax-test/src/test/java/com/netflix/astyanax/thrift/ThriftKeyspaceImplTest.java) test suites cover this "RegEx" solution

Reply:
this fix was canceled